modules/effects/arrow-effects/src/main/kotlin/arrow/effects/typeclasses/Fiber.kt

```diff
@@ -7,20 +7,20 @@ import arrow.effects.internal.CancelToken
  * [Fiber] represents the pure result of an [Async] data type
  * being started concurrently and that can be either joined or canceled.
  *
- * You can think of fibers as being lightweight threads, a fiber being a
+ * You can think of fibers as being lightweight threads, a Fiber being a
  * concurrency primitive for doing cooperative multi-tasking.
  */
 interface Fiber<F, A> {
 
   /**
    * Returns a new task that will await for the completion of the
-   * underlying fiber, (asynchronously) blocking the current run-loop
+   * underlying Fiber, (asynchronously) blocking the current run-loop
    * until that result is available.
    */
   val join: Kind<F, A>
 
   /**
-   * Triggers the cancellation of the fiber.
+   * Triggers the cancellation of the Fiber.
    *
    * @returns a task that trigger the cancellation upon evaluation.
    */
```

Comment:
NIT: Make these functions instead of fields even if they're cached. We don't have field-based APIs and it's harder to generate code for them.